controller_server:
  ros__parameters:
    goal_checker_plugins: ["goal_checker"]
    controller_plugins: ["FollowPath"]
    enable_stamped_cmd_vel: true

    goal_checker:
      plugin: "nav2_controller::SimpleGoalChecker"
      xy_goal_tolerance: 0.02
      yaw_goal_tolerance: 1.02
      stateful: True
    
    FollowPath:
      plugin: "twr_g2g_controller::G2GController"
      max_lin_vel: 0.2
      max_ang_vel: 1.5

      Kp_ang_vel: 2.0

      debug_info: true
      local_frame: "odom"